### ISSUE TITLE
HTTP File Upload getting started.

### ISSUE THREAD

**Issue by mailtosagar** [processone/ejabberd#2952]
Hi All,
I was reading about http file upload. 
My understanding so far is.
In ejabberd.yml
docroot: here files get saved when uploaded on server.
put_url: we can upload the file on this url.

What i am confused with when i am making a put request using postman . i am Receiving Cannot get any response from the server
![image](https://user-images.githubusercontent.com/16096109/61589561-55eecb80-abc9-11e9-8edf-901f659c187f.png)

here is my configurations in the ejabberd.yml file
 mod_http_upload:
    docroot: "/home/sagar-singh/Desktop/backups/chatting_server/TXTAPP_server/uploads"
    put_url: "http://@HOST@:5443/upload"
    max_size: 500000000 #500 MByte
    thumbnail: false
    custom_headers: start
      "Access-Control-Allow-Origin": "*"   
      "Access-Control-Allow-Methods": "PUT"  
      "Access-Control-Allow-Headers": "content-type"

right now i am running everything on the localhost. 
Also when i run the following url in browser .
"http://localhost:5443/upload" : i get "page isn't working.
"http://localhost:5280/upload" : i get "not found".

basically i am unable to find a path how i should get started with the file upload .
Please anyone help . It's urgent.  Thanks for the help in advance.
